pkg/swagger/generator/types.go

```diff
@@ -215,6 +216,24 @@ func (t *typeResolver) resolveFormat(schema *spec.Schema, isAnonymous bool, isRe
 	return
 }
 
+func (t *typeResolver) resolveExtensions(schema *spec.Schema, isAnonymous bool, isRequired bool) (returns bool, result resolvedType, err error) {
```

Comment:
- `resolveExtensions(schema *spec.Schema, isAnonymous, isRequired bool)` second and third args has the same `bool` type, the first type is optional
- return values are confused, what type of return value？`return value, ok` ?

Reply:
- fixed, ptal
- there are 3 return values: 
   - ```returns bool```: if the caller function should return. As the OpenAPI spec states, the type resolver will detect the result KCL type by multiple factors, and the type resolver will resolve specific keys step by step. For example if ```$ref``` appears as a key under a property, all the other keys will be ignored and the property's type will be resolved to the referenced model that the "$ref" defined. And in this PR's case, if ```{"x-kubernetes-int-or-string": "true"}``` appears, the type of the property will be ```int | str```, regardless the ```type``` and  ```format``` keys. So in cases above, the main type resolving process has no need to resolve other keys under the property spec and can directly return with the resolved type.
   - ```result resolvedType```: the result of the resolved type
   - ```err error```: if any error occurred in the sub process of resolving extensions, the error should be announced in the main type resolving process